addons/sys_core/fnc_muting.sqf

```diff
@@ -52,31 +47,29 @@ DFUNC(mutingPFHLoop) = {
                 if (!(_remoteTs3Id in ACRE_SPECTATORS_LIST)) then {
                     private _isRemotePlayerAlive = [_remoteUser] call FUNC(getAlive);
                     if (_isRemotePlayerAlive == 1) then {
-                        //PUSH(_playerIdList, _remoteTs3Id);
+                        //_playerIdList pushBack _remoteTs3Id;
 
                         // private _radioListRemote = [_remoteUser] call EFUNC(sys_data,getRemoteRadioList);
                         // private _radioListLocal = [] call EFUNC(sys_data,getPlayerRadioList);
                         // private _okRadios = [_radioListLocal, _radioListRemote, true] call EFUNC(sys_modes,checkAvailability);
                         if (_remoteUser distance _dynamicPos > 300) then {
-                            PUSH(_muting,_remoteUser);
+                            _muting pushBack _remoteUser;
                             _muted = 1;
+                            if (_remoteUser in GVAR(speakers)) then {
+                                _muted = 0;
+                            };
```

Comment:
Not sure if that's clearer, but could be written as
`_muted = [1, 0] select (_remoteUser in GVAR(speakers));`